docs/develop/cmodules.rst

```diff
@@ -70,9 +94,11 @@ and has a source file and a Makefile fragment with content as descibed above::
        └──usercmodule/
           └──cexample/
              ├── examplemodule.c
-             └── micropython.mk
+             ├── micropython.mk
+             └── usermod.cmake
+
 
-Refer to the comments in these 2 files for additional explanation.
+Refer to the comments in these 3 files for additional explanation.
```

Comment:
Maybe just remove the number completely, more maintainable.

Reply:
Agreed!